packages/smooth_app/lib/bottom_sheet_views/product_copy_view.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+
+class CopyToListView extends StatelessWidget {
+  const CopyToListView(
+    this._scrollController, {
+    this.callback,
+    @required this.children,
+  });
+
+  final ScrollController _scrollController;
+  final Function callback;
+  final List<Widget> children;
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      child: Container(
+        height: MediaQuery.of(context).size.height * 0.9,
```

Comment:
I have a personal problem with "bottom" sheets that take 90% of the screen. Wouldn't a page make more sense?
It's a personal opinion - you can ignore that comment.

Reply:
I just had a look how this looks like without a fixed height value, I didn't saw a big difference in the usability, but the header doesn't fit so well if you have the sheet with only one row of buttons.